tox.ini

```diff
@@ -14,8 +15,6 @@ deps =
 commands =
     unit: pytest {posargs}
     flake8: flake8 {posargs}
-    pylint: python setup.py lint
+    pylint: sh -c "git ls-files -co --exclude-standard '*.py' | grep -Ev '^(utils|roles/lib_(openshift|utils)/src)/' | xargs pylint"
```

Comment:
I like that this is more transparent than the setuptools integration, but I'm not sure this is the right answer. It'll skip over any files the user created/moved but hasn't done a `git add` on. That is what led me to the setuptools integration to be able to do better discovery...  Maybe we can leverage pytest for that instead?